robotbase/src/main/java/com/team2357/frc2022/commands/auto/AutoDriveCommand.java

```diff
@@ -0,0 +1,33 @@
+package com.team2357.frc2022.commands.auto;
+
+import com.team2357.lib.commands.CommandLoggerBase;
+import com.team2357.lib.subsystems.drive.FalconTrajectoryDriveSubsystem;
+
+public class AutoDriveCommand extends CommandLoggerBase {
+    double m_timeMillis;
+    double m_timeTotal;
+
+    public AutoDriveCommand (double timeMillis) {
+
+        addRequirements(FalconTrajectoryDriveSubsystem.getInstance());
+        m_timeMillis = timeMillis;
+    }
+
+    @Override
+    public void initialize() {
+        FalconTrajectoryDriveSubsystem.getInstance().driveVelocity(0.1, 0.0);
```

Comment:
I'd like to add a turn to this, but we can add that later as an optional argument.